spec/numberformat.html

```diff
@@ -30,9 +30,19 @@ <h1>SetNumberFormatDigitOptions ( _intlObj_, _options_, _mnfdDefault_, _mxfdDefa
           1. Set _intlObj_.[[MaximumSignificantDigits]] to _mxsd_.
         1. Else if _mnfd_ is not *undefined* or _mxfd_ is not *undefined*, then
           1. Set _intlObj_.[[RoundingType]] to ~fractionDigits~.
-          1. Let _mnfd_ be ? DefaultNumberOption(_mnfd_, 0, 20, _mnfdDefault_).
+          1. If _mxfd_ is *undefined*, then
+            1. Let _mnfdActualDefault_ be _mnfdDefault_.
+          1. Else,
+            1. Set _mxfd_ to ? ToNumber(_mxfd_).
+            1. If _mxfd_ is *NaN*, then throw a *RangeError* exception.
+            1. Set _mnfdActualDefault_ to min( max( _mxfd_, 0 ), _mnfdDefault_ ).
+          1. Let _mnfd_ be ? DefaultNumberOption(_mnfd_, 0, 20, _mnfdActualDefault_).
           1. Let _mxfdActualDefault_ be max( _mnfd_, _mxfdDefault_ ).
-          1. Let _mxfd_ be ? DefaultNumberOption(_mxfd_, _mnfd_, 20, _mxfdActualDefault_).
```

Comment:
We should keep this call to DefaultNumberOption, there's no need to *optimise* the spec for performance (by avoiding duplicate checks or reducing (unobservable) calls to ToNumber).